phpcs.ruleset.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<ruleset name="WP-API">
+	<description>Sniffs for the coding standards of the WP-API plugin</description>
+
+	<rule ref="WordPress-VIP">
```

Comment:
The ones I'm seeing that we could exclude:
* get_term_by is prohibited, please use wpcom_vip_get_term_by() instead.
* file_get_contents is highly discouraged, please use wpcom_vip_file_get_contents() instead
* switch_to_blog is not something you should ever need to do in a VIP theme context. Instead use an API (XML-RPC, REST) to interact with other sites if needed

Possibly also "Due to using Batcache, server side based client related logic will not work, use JS instead."